railties/test/application/action_controller_test_case_integration_test.rb

```diff
@@ -33,6 +33,7 @@ def time_zone
     RUBY
 
     remove_from_config '.*config\.load_defaults.*\n'
+    add_to_config "config.active_support.cache_format_version = 7.1"
```

Comment:
This unfortunately feels like the type of thing we'll inevitably have to update on new releases. Could we template like `Rails::VERSION::STRING.to_f` in here or is that overkill?

Reply:
I looked into this, and we can't simply use whatever the current `RAILS_VERSION` is because it may not be supported by AS::Cache:
https://github.com/rails/rails/blob/633eb5a9f4f7ed9e638ac5b6b9ee04f323bb61c0/activesupport/lib/active_support/cache.rb#L759-L763

Reply:
When #49624 is merged, this will be unnecessary (in terms of suppressing warnings), right?  If so, I think it would be better to leave this out.

Reply:
👍 Dropped.